git/githistory/rewriter.go

```diff
@@ -554,6 +554,7 @@ func (r *Rewriter) refsToMigrate() ([]*git.Ref, error) {
 		}
 
 		local = append(local, ref)
+		fmt.Fprintf(os.Stderr, "dx: b: to migrate=%q\n", ref.Refspec())
```

Comment:
Is this deliberate, or leftover from debugging, by any chance?

Reply:
That is indeed debugging output.  I'll strip it out.